flow/designs/src/mock-alu/defaults.mk

```diff
@@ -1,10 +1,15 @@
 export MOCK_ALU_WIDTH     ?= 64
 export MOCK_ALU_OPERATIONS?= ADD,SUB,AND,OR,XOR,SHL,SHR,SRA,SETCC_EQ,SETCC_NE,SETCC_LT,SETCC_ULT,SETCC_LE,SETCC_ULE,MULT
 
+export MOCK_REGISTERFILE_WIDTH     ?= 64
+export MOCK_REGISTERFILE_READ_PORTS ?= 8
+export MOCK_REGISTERFILE_WRITE_PORTS ?= 4
+export MOCK_REGISTERFILE_REGISTERS ?= 128
+
 export DESIGN_NAME            = MockAlu
 export DESIGN_NICKNAME        = mock-alu
 
-export VERILOG_FILES          = designs/src/mock-alu/*.v
+export VERILOG_FILES          = designs/src/mock-alu/*.sv designs/src/mock-alu/*.v
```

Comment:
Why are there .v & .sv?  Yosys doesn't support system verilog (without a plugin).  Are these all actually regular verilog?

Reply:
depending on which version of chisel you use, you get .v or .sv files.

The .sv language subset used by Chisel is supported by yosys.